exercises/affine-cipher/description.md

```diff
@@ -1,14 +1,12 @@
 # Description
 
-Create an implementation of the affine cipher,
-an ancient encryption system created in the Middle East.
+Create an implementation of the affine cipher, an ancient encryption system created in the Middle East.
 
-The affine cipher is a type of mono-alphabetic substitution cipher.
-Each character is mapped to its numeric equivalent, encrypted with
-a mathematical function and then converted to the letter relating to
-its new numeric value. Although all mono-alphabetic ciphers are weak,
-the affine cipher is much stronger than the atbash cipher,
-because it has many more keys.
+The affine cipher is a type of monoalphabetic substitution cipher.
+Each character is mapped to its numeric equivalent, encrypted with a mathematical function and then converted to the letter relating to its new numeric value.
+Although all monoalphabetic ciphers are weak, the affine cipher is much stronger than the atbash cipher, because it has many more keys.
```

Comment:
is this PR the place to discuss potential changes to the text, or does that go elsewhere because this PR was only meant to change formatting?

In particular, I'd like to call attention to the word "it"  - someone reading it might ask "which of the two ciphers is being referred to by 'it' here?"

and therefore, I have one of two suggestions.

```suggestion
Although all monoalphabetic ciphers are weak, the affine cipher is much stronger than the atbash cipher, because the former has many more keys.
```
```suggestion
Although all monoalphabetic ciphers are weak, the affine cipher is much stronger than the atbash cipher, because the former has many more keys than the latter.
```

Reply:
I think it is common for native speakers to infer properly, while perhaps English as a second and third language may miss it a bit.

But yeah, not quite right PR for this, but if I did this, it would be a different commit, so that if necessary it could be reverted without taking the formatting away.  (So it would not be squashed, only rebased.)  But I can not necessarily control how people merge this in, so I would almost prefer a different PR.

Reply:
I think making a different PR might also bring abut a "policy" to tend to more clearly tend to pronouns across the exercises?